diff --git a/include/clenabled/GRCLBase.h b/include/clenabled/GRCLBase.h
--- a/include/clenabled/GRCLBase.h
+++ b/include/clenabled/GRCLBase.h
@@ -17,9 +17,7 @@
 // You need to use the deprecated functions.  This #define makes the warning go away.
 // #define CL_USE_DEPRECATED_OPENCL_1_2_APIS
 // #define CL_VERSION_1_2
-#define CL_VERSION_2_2
-#define CL_HPP_TARGET_OPENCL_VERSION 210
-#define CL_TARGET_OPENCL_VERSION 220
+#define CL_HPP_TARGET_OPENCL_VERSION 200
 
 #define CL_HPP_ENABLE_SIZE_T_COMPATIBILITY
 #define CL_HPP_ENABLE_PROGRAM_CONSTRUCTION_FROM_ARRAY_COMPATIBILITY
@@ -46,7 +44,7 @@
 #if defined(__APPLE__) || defined(__MACOSX)
 #include <OpenCL/cl.hpp>
 #else
-#include <CL/cl.hpp>
 #endif
+#include <CL/opencl.hpp>
 
 #include "clSComplex.h"
diff --git a/lib/clview.cc b/lib/clview.cc
--- a/lib/clview.cc
+++ b/lib/clview.cc
@@ -2,17 +2,11 @@
 // Use CPP exception handling.
 // Note: If you include cl.hpp, the compiler just won't find cl::Error class.
 // You have to use cl2.hpp to get it to go away
-#define __CL_ENABLE_EXCEPTIONS
-#define CL_HPP_ENABLE_EXCEPTIONS
 // Disable the deprecated functions warning.  If you want to keep support for 1.2 devices
 // You need to use the deprecated functions.  This #define makes the warning go away.
 // #define CL_USE_DEPRECATED_OPENCL_1_2_APIS
-#define CL_VERSION_2_2
-#define CL_HPP_TARGET_OPENCL_VERSION 210
-#define CL_TARGET_OPENCL_VERSION 220
+#define CL_HPP_TARGET_OPENCL_VERSION 200
 
-#define CL_HPP_ENABLE_SIZE_T_COMPATIBILITY
-#define CL_HPP_ENABLE_PROGRAM_CONSTRUCTION_FROM_ARRAY_COMPATIBILITY
 /*
 #ifdef CL2_FOUND
        // #pragma message("Compiling with CL2 support")
@@ -33,7 +27,7 @@
 #if defined(__APPLE__) || defined(__MACOSX)
 #include <OpenCL/cl.hpp>
 #else
-#include <CL/cl.hpp>
+#include <CL/opencl.hpp>
 #endif
 
 #include <iostream>
